charts/kubewarden-controller/Chart.yaml

```diff
@@ -31,7 +33,7 @@ annotations:
   catalog.cattle.io/display-name: Kubewarden # Only for Charts with custom UI
   catalog.cattle.io/os: linux # this means linux only, other choice here is "windows". For charts that support both, don't add this annotation
   # optional ones:
-  catalog.cattle.io/auto-install: kubewarden-crds=1.3.1 # Similar to requires but auto-installed, not manually installed. Accepts `match`, or a specific version.
+  catalog.cattle.io/auto-install: kubewarden-crds=1.4.0 # Similar to requires but auto-installed, not manually installed. Accepts `match`, or a specific version.
   catalog.cattle.io/provides-gvr: "policyservers.policies.kubewarden.io/v1" # Declare that this chart provides a type, which other charts may use in `requires-gvr`. Only add to parent, not CRD chart.
```

Comment:
From Rancher docs:
> We only support adding in one CRD, if your chart supports multiple GVRs just pick one of them that will be consistent across all these labels. This chart will not be automatically installed, but instead the user will have to install it themselves before the current chart can be installed.

So that means we can't add `policyreports.wgpolicyk8s.io` and clusterpolicyreports. But no problem (notice how we only have policyservers and not admissionpolicies either).